py-deploy/tldeploy/core.py

```diff
@@ -107,7 +109,7 @@ def deploy_networks(web3, network_settings, currency_network_contract_name=None)
             web3,
             exchange_address=exchange.address,
             currency_network_contract_name=currency_network_contract_name,
-            **network_setting,
+            **network_setting
```

Comment:
Can you please try to add a pyproject.toml like we have in the relay server repo to force black's target version to be python 3.6?

It should look like:
```
[tool.black]
target-version = ['py36']
```

Then you wouldn't introduce those diffs.

Reply:
I added it in https://github.com/trustlines-protocol/contracts/pull/225 to avoid having to rebase / resolve conflicts.